### PR TITLE
feature-task-14 | task- even number count using block proc and lambda

### PR DESCRIPTION
**Pull request by SaurabhJ-weboniselab**

### CODE REVIEW

**Comment by eliza-abraham** [comment]
use proper branch name

File: assignment_1.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+arr = [2, 3, 4, 5, 6, 7]
+count = 0
+arr.each { |x| count += 1 if x.even? }
+
+puts count
+
+count = 0
+
+lmda = ->(var) { var.each { |x| count += 1 if x.even? } }
```
Comment (eliza-abraham) [inline assignment_1.rb:11]: use meaningful name for the lambda

File: assignment_1.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+lmda.call(arr)
+
+count = 0
+my_proc = proc { |var| var.each { |x| count += 1 if x.even? } }
```
Comment (eliza-abraham) [inline assignment_1.rb:15]: use meaningful name for the proc

File: assignment_1.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+arr = [2, 3, 4, 5, 6, 7]
+count = 0
+arr.each { |x| count += 1 if x.even? }
+
+puts count
+
+count = 0
+
+lmda = ->(var) { var.each { |x| count += 1 if x.even? } }
+lmda.call(arr)
```
Comment (pritam-webonise) [inline assignment_1.rb:12]: did you check the out of it?

File: assignment_1.rb (hunk 2 above)
Comment (pritam-webonise) [inline assignment_1.rb:15]: what is my_proc here?

**Pull request closed without merging**